src/platforms/common/data-management/event-grouping/grouping-enhancements.mdx

```diff
@@ -92,30 +80,28 @@ stack.function:malloc      -group
 
 alias: `package`
 
-Matches on a package in a stack trace. The package is the container that contains a function or module.
-This is a `.jar`, a `.dylib` or similar. The same matching rules as for `path` apply (For example, this is typically an absolute path).
+Matches on a package in a stack trace. The package is the container that contains a function or module. This is a `.jar`, a `.dylib` or similar. The same matching rules as for `path` apply. For example, this is typically an absolute path.
 
 ```discover {tabTitle:Enhancement Rules}
 stack.package:**/libcurl.dylib -group
 ```
 
 ### `app`
 
-Matches on the current state of the in-app flag of a stack trace frame. `yes`
-means the frame is in app, `no` means it's not.
+Matches on the current state of the in-app flag of a stack trace frame. `yes` means the frame is in-app, `no` means it's not.
 
 </DefinitionList>
 
 ## Actions
 
-There are two types of **actions**: flag setting and setting variables.
+There are two types of **actions**: flag and variables setting.
 
-- **flag**: flags identify the action to be taken if all matchers match. A flag needs to be prefixed with `+` to set it or `-` to unset it. If this expression is prefixed with a `^`, it applies to frames above the frame (toward the crash). If prefixed with `v` it applies to frames below the frame (away from the crash). For instance, `-group ^-group` removes the matching frame and all frames above it from the grouping.
+- **flag**: Flags identify the action to be taken if all matchers match. A flag needs to be prefixed with `+` to set it or `-` to unset it. If this expression is prefixed with a `^`, it applies to frames above the flag (toward the crash). If prefixed with `v` it applies to frames below the flag (away from the crash). For instance, `-group ^-group` removes the matching frame and all frames above it from the grouping.
```

Comment:
@rbisol I changed the word "frame" to "flag" in a few places because I assumed "frame" was used incorrectly. Will you confirm?

Reply:
Frame in this context was the correct word but unclear. I suggest changing it to "matching frame":

```suggestion
- **flag**: Flags identify the action to be taken if all matchers match. A flag needs to be prefixed with `+` to set it or `-` to unset it. If this expression is prefixed with a `^`, it applies to frames above the matching frame (toward the crash). If prefixed with `v` it applies to frames below the matching frame (away from the crash). For instance, `-group ^-group` removes the matching frame and all frames above it from the grouping.
```